SalesforceCommerceCloud/pwa-kit
# W-12627197 - Indicate live regions for dynamically changing content

# Description

This PR updates the code to make sure dynamic content is visible to screen reader in cart product-item and order summary 


https://github.com/SalesforceCommerceCloud/pwa-kit/assets/52219283/28b05625-0870-4667-bdc0-eb2ae4aae142





# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- item-variant.jsx

# How to Test-Drive This PR

- Checkout code
- Run Retail App template
- Add a few items in cart
- Go to Cart Page.
- Open VoiceOver (Cmd + f5)
- Change the quantity
- Observe the screen reader is now reading out loud the changes in quantity, prices and order summary section

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
